public/javascripts/Choropleth.js

```diff
@@ -155,6 +155,44 @@ function Choropleth(_, $, turf) {
             })
                 .addTo(map);
         });
+
```

Comment:
And a comment here saying something like "logs when the user clicks to select their own neighborhood" or something, maybe even saying what "activity" would be logged exactly.